medicines/doc-index-updater/Cargo.toml

```diff
@@ -20,7 +20,7 @@ md5 = "0.7.0"
 redis = { version = "0.15.1", features = ["tokio-rt-core"] }
 regex = "1.3.1"
 reqwest = { version = "0.10.4", features = ["json"] }
-mhra_products_search_client = "0.0.1"
+search_client =  { git = "https://github.com/MHRA/products", branch = "createable-deleteable-into-search-crate" }
```

Comment:
I think this is the problem that [Cargo workspaces][1] solve. It seems weird for a repo to reference itself like this.

[1]: https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html